src/styles/main.scss

```diff
@@ -1,3 +1,56 @@
 body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: widthCalendar($cell-size, $dayGap, $padding);
+  padding: 30px $padding;
+  gap: $dayGap;
+  margin: 0 auto;
+
+  &__day {
+    box-sizing: border-box;
```

Comment:
This rule will be applied for all elements in your project, will be better to set it using `*` selector